prow/gerrit/adapter/adapter.go

```diff
@@ -464,17 +445,21 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 	switch change.Status {
 	case client.Merged:
 		var postsubmits []config.Postsubmit
-		for attempt := 0; attempt < inRepoConfigRetries; attempt++ {
-			postsubmits, err = c.inRepoConfigCacheHandler.GetPostsubmits(trimmedHostPath, func() (string, error) { return baseSHA, nil }, func() (string, error) { return change.CurrentRevision, nil })
-			// Break if there was no error, or if there was a merge conflict
-			if err == nil || strings.Contains(err.Error(), "Merge conflict in") {
-				break
+		if c.config().InRepoConfigEnabled(cloneURI) {
+			for attempt := 0; attempt < inRepoConfigRetries; attempt++ {
+				postsubmits, err = c.inRepoConfigCacheHandler.GetPostsubmits(cloneURI, func() (string, error) { return baseSHA, nil }, func() (string, error) { return change.CurrentRevision, nil })
```

Comment:
We're now using the prefixed `cloneURI` rather than the unprefixed `trimmedHostPath`. I can't tell if that is intentional or if it is safe to change that.

Reply:
yeah it should be safe. The idea from this PR is to assume that other than user configured, `org`, `cloneURI` that are passed around must include the `https://` prefix